### PR TITLE
Fix KonnectorAction CTA

### PR DESCRIPTION
**Pull request by ptbrowne**
Following the selector refactor, withAppUrls was broken, this prevented
KonnectorAction to correctly work.

### CODE REVIEW

File: src/ducks/settings/AccountsSettings.jsx [hunk 1]
```diff
@@ -130,7 +130,7 @@ class AccountsSettings extends Component {
 }
 
 const mapStateToProps = state => ({
-  collectUrl: getAppUrlById(state, 'io.cozy.apps/collect')
+  collectUrl: getHomeURL(state)
```
Comment (drazik) [inline src/ducks/settings/AccountsSettings.jsx]: Does it require lot of changes to rename this `collectUrl` to `homeUrl`?

File: src/ducks/apps/selectors.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ import keyBy from 'lodash/keyBy'
 import { createSelector } from 'reselect'
 import { queryDataSelector } from 'selectors'
 
+export const getAppURL = app => (app ? app.links.related : null)
```
Comment (drazik) [inline src/ducks/apps/selectors.js]: We should be more safe with `app => (app && app.links ? app.links.related : null)`

File: src/ducks/settings/AccountsSettings.jsx (hunk 1 above)
Comment (ptbrowne) [reply src/ducks/settings/AccountsSettings.jsx]: This component did not use collectUrl anymore :tada:

File: src/ducks/apps/selectors.js (hunk 2 above)
Comment (ptbrowne) [reply src/ducks/apps/selectors.js]: Done thanks

**Pull request merged**